src/server/transaction.h

```diff
@@ -515,6 +518,8 @@ class Transaction {
   // subject to uncontended keys.
   bool ScheduleInShard(EngineShard* shard, bool execute_optimistic);
 
+  static void ScheduleBatchInShard();
```

Comment:
Please comment on what it does and what the motiviation is, we'll be puzzling in a few months